manage/config-dialog.js

```diff
@@ -286,7 +316,33 @@ function configDialog(style) {
   }
 
   function updateVarOnChange() {
-    this.va.value = this.type !== 'checkbox' ? this.value : this.checked ? '1' : '0';
+    if (this.type === 'number') {
+      this.value = this.va.value = clampValue(this.value, this.va);
```

Comment:
I think resetting the value while the user is typing is confusing. Maybe we should only reset the input value on the `blur` event?
![test](https://user-images.githubusercontent.com/1324510/45734409-08a37580-bc17-11e8-87da-9d143338e736.gif)

Reply:
Or show a small tooltip that informs the value was reset.

Reply:
I changed it to only clamp the value on blur, but now the raw value is saved in the live preview... I guess that's okay because it changes once the input is blurred.

Reply:
Actually, if you immediately close the popup, the invalid value remains.

Reply:
Even if we use a timer to reset the value, it'll remain if the user closes the dialog/popup.

Reply:
I think this is bad, and worse than having the value reset forcefully while typing. In colorpicker I simply didn't use an invalid value and nothing was reported upstream, the input element was set into an invalid state which was reddened via CSS. Can you do the same here?

Reply:
Yes, this is the best idea... now implemented.